login.css

```diff
@@ -0,0 +1,34 @@
+#body-wrapper{
+	width: 1000px;
+	top:0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	margin: auto;
+}
+.formBox{
+	width: 100%;
+  height: 640px;
```

Comment:
A good practice to get into is properly formatting your code, whether it be HTML, CSS or JS. This helps improve overall code legibility and aids in code base consistency and standardization.